pages/api/v1/activity-feed/subscribed-posts.ts

```
@@ -0,0 +1,335 @@
+							? {
+									id: topic_id,
+									name: getTopicNameFromTopicId(topic_id)
+							  }
+							: topicFromType,
+						track_no: !isNaN(trackNumber) ? trackNumber : null,
+						type: type || ProposalType.REFERENDUM_V2,
+						user_id: data?.user_id || 1
+					};
+					await getContentSummary(post, network, true);
+
+					delete post?.content;
```

Biome

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)